src/media/css/feed/brands.styl

```diff
@@ -67,7 +67,27 @@
 }
 
 .brand-landing-header-desktop {
+    margin-top: 22px;  // To get 40px. Combine with 18px #page padding.
     display: none;
+    position: relative;
+    z-index: 1;
+
+    h1 {
+        desktop_mega();
+        margin: 20px 0 40px;
+        text-align: center;
+    }
+}
+
+.brand-landing-curve {
+    background: rgb(250, 250, 250);
+    border-radius: 50% 50% 0 0;
```

Comment:
it looks curvy to me, how much smoother would it be able to get? i don't think it's worth the extra KBs though